WordPress/gutenberg
# Mobile - Fix line-height issue on Android

- Gutenberg Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4398

Fixes https://github.com/WordPress/gutenberg/issues/37168

## Description
This PR fixes an issue where the line-height value wouldn't be set after a font-size value was changed. Now it'll store the current line-height value and it'll be refreshed after any font size change.

**Note**: This feature is still under the `__DEV__` flag only available running metro.

## How has this been tested?

**Precondition**: Have a block-based theme activated.

- Open the WordPress Android app with metro running and setting the `localGutenbergMobilePath` path.
- Add a `Heading` block
- Type some text, with enough characters to fill in at least two lines
- **Expect** the line height to be shown correctly

## Screenshots 
Before | After
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/147077721-ac6b9ade-ddcf-4e1e-9404-736c7e27449f.png" width="200" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/147077730-697f8bc1-665f-456e-b19c-33d0c642fed2.png" width="200" /></kbd>

## Types of changes
Bug fix

## Checklist:
